### PR TITLE
[12.0][MIG] account_mass_reconcile_by_purchase_line

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

**Comment by HviorForgeFlow** [comment]
FYI @mreficent rebase done

File: account_mass_reconcile_by_purchase_line/README.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+using the PO Line and Product as key fields. This type of
+reconciliation is to be used in the context of the Perpetual Inventory
+accounting system, with the accrual account '*Goods Received Not Invoiced*'.
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to 'Accounting / Adviser / Mass Automatic Reconcile'.
```
Comment (AaronHForgeFlow) [inline account_mass_reconcile_by_purchase_line/README.rst]: Invoicing/Invoicing/Accounting/Actions/MassAutomaticReconcile

**Review by AaronHForgeFlow — COMMENTED** [review]
LGTM minor comment

**Review by AaronHForgeFlow — APPROVED** [review]
Not bad :+1:

**Review by AdriaGForgeFlow — APPROVED** [review]
Code review :+1:

**Comment by AaronHForgeFlow** [comment]
Ready to merge

**Review by HviorForgeFlow — APPROVED** [review]

**Review by JoanSForgeFlow — APPROVED** [review]

**Review by AdriaGForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — APPROVED** [review]

**Comment by AaronHForgeFlow** [comment]
Ready to merge IMVHO

**Comment by AaronHForgeFlow** [comment]
It seems there are lint errors in the base module and because of that this is failing

**Comment by MiquelRForgeFlow** [comment]
@dreispt @AaronHForgeFlow error is solved. please, could you merge?

**Pull request merged**